DESCRIPTION

```diff
@@ -13,6 +13,7 @@ Imports:
     config,
     data.table,
     dplyr,
+    GAMBLR.data,
```

Comment:
You also need to point where to find this package by adding the Remote key before Encoding

Reply:
Oh, I forgot this! Thanks

Reply:
Done